snekde/snekde.py

```diff
@@ -13,17 +13,16 @@
 # General Public License for more details.
 #
 
-import sys
 import argparse
-import time
 import curses
-import threading
 import serial
 import serial.tools.list_ports
-
-from pathlib import Path, PurePath
+import sys
+import threading
+import time
 
 from curses import ascii
+from pathlib import Path, PurePath
```

Comment:
Is this following a convention for ordering imports in python programs?

Reply:
I just sorted them alphabetically.

https://www.python.org/dev/peps/pep-0008/#imports
> Imports should be grouped in the following order:
>
>   Standard library imports.
>   Related third party imports.
>   Local application/library specific imports.
> 
> You should put a blank line between each group of imports.